src/service/video-manager-impl.js

```diff
@@ -172,6 +174,40 @@ export class VideoManager {
     }
   }
 
+
+  /**
+   * Install the necessary listeners to be notified when the user changes
+   * the orientation of their device
+   *
+   * @param {VideoEntry} entry
+   * @private
+   */
+  maybeInstallOrientationObserver_(entry) {
+    // TODO(@wassgha): Remove this later. For now, the orientation observer
```

Comment:
let's replace the `TODO` with a comment that we only needs this when `auto-fullscreen` is enabled. Can't think of any other cases we may need this observer.